collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceVisibilityHandler.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2017 Nafundi
```

Comment:
Probably I should remove it? This class look nothing like its old version.

Reply:
In this case it doesn't particularly matter since Nafundi can grant copyright to ODK. Maybe it would have been preferable to update it to 2021 ODK but we're not consistent with these headers at all anyway.